### PR TITLE
Multiple commands

### PR DESCRIPTION
**Pull request by LamprosPitsillos**
Allow for multiple commands without chaining them with the `|` operator.

### CODE REVIEW

File: lua/attempt/interface.lua [hunk 1]
```diff
@@ -35,8 +35,10 @@ function M.run(bufnr)
     return
   end
   print('\n') -- Prevent output from overlapping with existing msgs
-  if type(config.opts.run[file_entry.ext]) == 'string' then
-    vim.cmd(config.opts.run[file_entry.ext])
+  if type(config.opts.run[file_entry.ext]) == 'table' then
+		for _, cmd in pairs(config.opts.run[file_entry.ext]) do
+			vim.cmd(cmd)
+		end
   else
```
Comment (m-demare) [inline lua/attempt/interface.lua:42]: I'd add the `string` case, with the same behaviour as before, to mantain backwards compatibility

File: lua/attempt/config.lua [hunk 2]
```diff
@@ -3,41 +3,44 @@ local M = {}
 local unix = vim.fn.has 'unix' == 1
 local path_separator = unix and '/' or '\\'
 
+-- local cpp_compiler = "g++"
+-- local c_compiler = "gcc"
```
Comment (m-demare) [inline lua/attempt/config.lua:7]: I'd remove these comments, unless you want to submit both changes in this PR (which is fine by me)

**Review by m-demare — CHANGES_REQUESTED** [review]
Thanks for the contribution! I left a couple of comments, and also please use 2 space indentation, for consistency with the rest of the code ;)

**Review by m-demare — COMMENTED** [review]
Great! Please fix indentation (for consistency with the rest of the project) and I'll merge it
You can fix it in vim (in all open buffers) using `:bufdo %sm/\t/  /g` (that's 2 spaces between the `/  /`, for some reason github renders it as one)

**Comment by LamprosPitsillos** [comment]
> Great! Please fix indentation (for consistency with the rest of the project) and I'll merge it You can fix it in vim (in all open buffers) using `:bufdo %sm/\t/ /g` (that's 2 spaces between the `/ /`, for some reason github renders it as one)

It does look better , with spaces , you are right ! I was just using stylua.

**Review by m-demare — APPROVED** [review]

**Comment by m-demare** [comment]
Thank you for the contribution!

**Pull request merged**